### DISCUSSION TITLE
Host and Port in production mode

### DISCUSSION THREAD

**Discussion by mihail727** [nuxt/nuxt#16004] [Questions] [answered]
Good afternoon, please tell me how to change Host and Port in production mode?
![image](https://user-images.githubusercontent.com/25690524/141458004-e29686b6-9095-49f0-b64a-d283428830b6.png)
![image](https://user-images.githubusercontent.com/25690524/141458032-349be08a-b597-458a-8116-639b7a006b83.png)

**Comment by HealerNguyen** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/29653304/141471278-89f11fa5-7d63-488a-8ded-df659268624d.png)
u can setting like this
But if u run product in ubuntu server, u can put HOST, PORT in your service file (eg: yourweb.service) and run with systemd, or env file if u running with docker

**Reply by mihail727** [reply]
thx, that works for me!

**Reply by jboada** [reply]
Hi @HealerNguyen ,

Is there a way to use this solution in Ubuntu + nginx + pm2?

Best Regards,
JB

**Reply by HealerNguyen** [reply]
hi, you can refer to nuxt 3 document https://nuxt.com/docs/getting-started/deployment, it contains node and pm2 guideline
- after you run your project with pm2 success, you can config nginx like that:
- 
![image](https://github.com/nuxt/nuxt/assets/29653304/2964bc3e-a4ef-4c33-87a9-106560f69f07)

**Comment by cryptic-dev** [comment]
Anyone still having issues, added a .env with PORT=9000 in nuxt 3

**Reply by zingfeng** [reply]
It worked!
add file .env 
-> run `nuxt build` then `nuxt start`
![image](https://user-images.githubusercontent.com/85675721/210991071-bae80562-a0c4-4612-be8f-4f092d3235b8.png)
